CONTRIBUTING.md

```diff
@@ -1,54 +1,87 @@
 
-Please read this if you intend to contribute to the project.
+This document describes a very simple process suitable for most projects in the Vert.x ecosystem. Projects are encouraged to adopt this whether they are hosted in the Vert.x Organization or not.
 
-## Legal stuff
+The goal of this document is to create a contribution process that:
 
-Apologies in advance for the extra work required here - this is necessary to comply with the Eclipse Foundation's
-strict IP policy.
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how contributors can be involved in decision making.
 
-Please also read [this](http://wiki.eclipse.org/Development_Resources/Contributing_via_Git)
 
-In order for any contributions to be accepted you MUST do the following things.
+## Vocabulary
 
-* Sign the [Eclipse Foundation Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
-To sign the Eclipse CLA you need to:
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Committer** is a subset of contributors who have been given write access to the repository.
 
-  * Obtain an Eclipse Foundation userid. Anyone who currently uses Eclipse Bugzilla or Gerrit systems already has one of those.
-If you don’t, you need to [register](https://dev.eclipse.org/site_login/createaccount.php).
 
-  * Login into the [projects portal](https://projects.eclipse.org/), select “My Account”, and then the “Contributor License Agreement” tab.
+# Logging Issues
 
-* Add your github username in your Eclipse Foundation account settings. Log in it to Eclipse and go to account settings.
+Log an issue for any problem you might have. When in doubt, log an issue, any additional policies about what to include will be provided in the responses. The only exception is security dislosures which should be sent privately.
 
-* "Sign-off" your commits (NOTE! This is only necessary if contributing to the vertx-core project)
+Committers may direct you to another repository, ask for additional clarifications, and add appropriate info before the issue is addressed.
 
-Every commit you make in your patch or pull request MUST be "signed off".
+For questions that are not an issue with the code per se, it is recommended that you refer to the [community group](https://groups.google.com/forum/#!forum/vertx) since it allows the whole community to help instead of just the contributors and commiters.
 
-You do this by adding the `-s` flag when you make the commit(s), e.g.
 
-    git commit -s -m "Shave the yak some more"
+# Contributions
 
-## Making your changes
+Any change to resources in this repository must be through pull requests. This applies to all changes to documentation, code, binary files, etc. Even long term committers must use pull requests.
```

Comment:
Is that true currently? My understanding is that commiters can push to master "simple" changes.